website/versioned_docs/version-0.62/rnm-getting-started.md

```diff
@@ -17,7 +17,7 @@ Make sure you have installed all the [development dependencies](https://microsof
 Remember to call `react-native init` from the place you want your project directory to live.
 
 ```
-npx react-native init <projectName> --version 0.61.5
+npx react-native init <projectName> --version 0.62
```

Comment:
The 0.62 update for macOS is underway but not yet released (see [releases](https://github.com/microsoft/react-native-macos/releases)). So I don't think these instructions will give you what you want (yet), right?

Reply:
oh thanks you're right,

Reply:
i would this PR still waiting the new release for macOS installation. @chrisglein thank you!.

Reply:
Fast forward a year and now macOS is on 0.63 and working towards 0.64. So really we should bump this to 0.63 at the least (https://github.com/microsoft/react-native-macos/releases/tag/v0.63.37).
@HeyImChris FYI

Reply:
Oh, just realized this is editing the versioned docs. So this should be okay. Although looking at the versioned docs today, I think this got fixed?
https://microsoft.github.io/react-native-windows/docs/0.62/rnm-getting-started
https://microsoft.github.io/react-native-windows/docs/0.63/rnm-getting-started
So I think we can close out this PR as this all got sorted out.